### PR TITLE
Isogram implementation. 

### PR DESCRIPTION
**Pull request by Average-user**

### CODE REVIEW

File: exercises/isogram/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- lumberjacks
+- background
+- downstream
+- six-year-old
+
+The word *isograms*, however, is not an isogram, because the s repeats.
+
+# Implementation notes
+
+The Hamming distance is only defined for sequences of equal length. This means
```
Comment (verdammelt) [inline exercises/isogram/README.md]: This seems out of place? Copy-and-paste error?

File: exercises/isogram/example.lisp [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+(in-package #:isogram)
+
+(defun str-list (s)
+  (assert (stringp s) (s))
+  (coerce (string-upcase s) 'list))
+
+(defun is-isogram-helper (xs)
+  (if (null xs) T
+    (and (not (and (find (car xs) "ABCDEFGHIJKLMNOPQRSTUVWXYZ")
```
Comment (verdammelt) [inline exercises/isogram/example.lisp]: Could `#'alpha-char-p` be helpful here?

File: exercises/isogram/README.md (hunk 1 above)
Comment (Average-user) [reply exercises/isogram/README.md]: Prob, but wasn't the READMEs generated automatically anyway?

File: exercises/isogram/example.lisp (hunk 2 above)
Comment (Average-user) [reply exercises/isogram/example.lisp]: Maybe, but has problems like:
```Lisp
* (alpha-char-p #\ñ)        

T
```

**Pull request merged**